middle-server/src/controllers/prometheus/getAssignedTo.ts

```
@@ -30,15 +31,15 @@ export const getAssignedTo = async (req: Request, res: Response) => {
 
 export const getAssignedToLogic = async (swarmsBountyId: string): Promise<{ statuscode: number; data: any }> => {
   try {
-    const assignedTo = await DocumentationModel.findOne({ swarmBountyId: swarmsBountyId });
+    const assignedTo = await TodoModel.findOne({ bountyId: swarmsBountyId });
     console.log(assignedTo);
-    if (assignedTo && assignedTo.assignedTo) {
+    if (assignedTo && assignedTo.assignees) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)